src/AbstractTooltip/index.tsx

```diff
@@ -49,6 +49,10 @@ export const AbstractTooltip: React.FC<Props> = ({
   const disableAnimations =
     disableAnimationsFromProvider || forceVisibleForTestingOnly;
 
+  if (!props.content) {
+    return children ?? null;
```

Comment:
Can we switch this to also check for empty array? Right now I think this would still show up for
```jsx
<>
  {false && ...}
  {false && ...}
</>
```

Reply:
Can we also change this on `Card`? I think it our graph integrations page this is causing a bunch of extra margins

Reply:
I'm going to stick with using the `disabled` prop and I feel that an empty card should continue to render itself. In this case, this isn't an empty array, it's a Fragment that it's own children.

This is a different case because rendering content with or without the tooltip is not an easy change, as you know because you requested this 😃

Reply:
Ah sorry this was a different thing. We add padding to the bottom of the card only if the card has children, I think we use this same style check. We should fix it to check for empty arrays as well

Reply:
> Ah sorry this was a different thing. We add padding to the bottom of the card only if the card has children, I think we use this same style check. We should fix it to check for empty arrays as well

Ah I better understand now. That makes sense.